src/styles/global.css

```
@@ -307,3 +306,7 @@ dialog {
     }
   
 }
+
+.font-cinzel {
+  text-rendering: optimizeLegibility;
```

Stylelint (17.12.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)